Rmd/Sample_QC.Rmd

```diff
@@ -10,7 +10,7 @@ params:
   nmr_threshold: 100000    # 10% of 1M reads for TempOSeq = 100,000; 10% of 10M reads for RNA-Seq = 1,000,000.
   align_threshold: 0.5     # 50% alignment rate for TempO-Seq Experiments
   gini_cutoff: 0.95
-  exp_groups:
+  groups:
```

Comment:
I had renamed this one because groups is a built in something or other in R

Reply:
Also if you renamed it here there's about a billion other spots where it would have to get renamed

Reply:
oh yeah? Ok I'll fix it in the config then

Reply:
I guess it never made it to default? That's weird

Reply:
> I guess it never made it to default? That's weird

I think it did though...

Reply:
Ah sorry looks like it was a mistake on my end, I see that the default config has it right. I will undo this change.

Reply:
Maybe we should write a script to test whether the config is valid. Is that in the issues list already? I can't recall